sql/mob_pools.sql

```diff
@@ -628,7 +628,7 @@ INSERT INTO `mob_pools` VALUES (572,'Bukhis','Bukhis',240,0x00005609000000000000
 INSERT INTO `mob_pools` VALUES (573,'Bukki','Bukki',165,0x0000BE0600000000000000000000000000000000,4,4,7,240,100,0,0,0,0,2,0,0,0,3,0,0,2,0,0,165,165);
 INSERT INTO `mob_pools` VALUES (574,'Bullbeggar','Bullbeggar',188,0x0000A00100000000000000000000000000000000,1,1,7,200,100,0,1,0,1,0,0,0,139,133,0,0,0,0,0,188,188);
 INSERT INTO `mob_pools` VALUES (575,'Bulldog_Bats','Bulldog_Bats',47,0x0000040100000000000000000000000000000000,1,1,11,240,100,0,1,0,0,0,0,0,0,3,0,0,0,0,0,47,47);
-INSERT INTO `mob_pools` VALUES (576,'Bullheaded_Grosvez','Bullheaded_Grosvez',334,0x0000700200000000000000000000000000000000,2,2,1,480,100,0,1,0,1,0,0,0,0,3,0,0,0,0,0,334,334);
+INSERT INTO `mob_pools` VALUES (576,'Bullheaded_Grosvez','Bullheaded_Grosvez',334,0x0000700200000000000000000000000000000000,2,2,1,480,100,0,1,0,1,18,0,0,0,3,0,0,0,0,0,334,334);
```

Comment:
mobtype 18 would be a BCNM(16) + NM (2). Is this what you're going for here?
Here's the ENUMs for it:
    MOBTYPE_NORMAL      = 0x00,
    MOBTYPE_0X01        = 0x01, // available for use
    MOBTYPE_NOTORIOUS   = 0x02,
    MOBTYPE_FISHED      = 0x04,
    MOBTYPE_CALLED      = 0x08,
    MOBTYPE_BATTLEFIELD = 0x10, // 16
    MOBTYPE_EVENT       = 0x20 // 32

Reply:
It's not a BCNM, it's a popped mob in Carpenter's Landing.  In this case, 2 is all I need.